evm_body.yaml

```diff
@@ -1547,24 +1555,28 @@ eth_call_550_gas:
           default: eth_call
         params:
           type: array
-          minItems: 1
-          maxItems: 1
+          minItems: 3
+          maxItems: 3
           items:
             oneOf:
```

Comment:
@SahilAujla i don't think we need this `oneOf`

Reply:
@elanh we need it because Readme can't render multi-type arrays, so we use this workaround. 

For context here's an example of a page which doesn't use `OneOf`: https://test-base.readme.io/reference/eth-estimategas